server/helpers/identification.js

```diff
@@ -0,0 +1,25 @@
+import { UserRole, User, Role } from '../database/models';
+import responseMessage from './responseMessage';
+
+const getUserId = async (email, response) => {
+  try {
+    const { dataValues } = await User.findOne({ where: { email } });
+    return dataValues.id;
+  } catch (error) {
+    responseMessage(response, 404, { message: 'Account not found', error });
```

Comment:
I don't think you need to return the error object to the user. The message already tells the user what has happened.

Reply:
Done